cleanup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+rm -rf .idea
+./gradlew clean
+rm -rf .gradle
+rm -rf build
+rm -rf */build
+rm -rf demos/appyx-interactions/iosApp/iosApp.xcworkspace
+rm -rf demos/appyx-interactions/iosApp/Pods
+rm -rf demos/appyx-interactions/iosApp/iosApp.xcodeproj/project.xcworkspace
+rm -rf demos/appyx-interactions/iosApp/iosApp.xcodeproj/xcuserdata
```

Comment:
Is this script required, or was it just for your testing? is this something that needs to be added as part of the gradle clean task?

Reply:
Jetbrains' iOS template contains this file. Yeah, I believe having it be part of the gradle clean task would be useful. It's helpful when things aren't building properly on iOS.